ansible/config_sonic_basedon_testbed.yml

```diff
@@ -279,7 +279,7 @@
   - block:
       - name: Create new dictionary for my slot
         set_fact:
-          new_asic_topo_config : "{{ new_asic_topo_config | default( { slot_num  : asic_topo_config.slot0 }) }}"
+          new_asic_topo_config : "{{ new_asic_topo_config | default( { slot_num  : asic_topo_config[slot_num] }) }}"
```

Comment:
why do we need this change?

Reply:
Hi @arlakshm, I have removed this change from the current PR. I will talk with the team and if the template change is needed, I will go ahead and create a separate PR.

Reply:
An update on this, I had a conversation with Song and we have concluded that the config change is not needed.